### ISSUE TITLE
[Bug] NU1608 warnings after updating AndroidX nuget packages

### ISSUE THREAD

**Issue by MagicAndre1981** [xamarin/Xamarin.Forms#15839]
### Description

I commented about this in the last PR that updated the dependency constraint, after updating to latest AndroidX libs I get [NU1608 warnings](https://learn.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu1608)

### Steps to Reproduce

1. install latest AndroidX updates
2. compile your app

### Expected Behavior

now warnings

### Actual Behavior

NU1608 warnings:

```
NU1608: Detected package version outside of dependency constraint: Xamarin.Forms 5.0.0.2612 requires Xamarin.AndroidX.Lifecycle.LiveData (>= 2.3.1.1 && <= 2.6.1.1) but version Xamarin.AndroidX.Lifecycle.LiveData 2.6.2.2 was resolved.
NU1608: Detected package version outside of dependency constraint: Xamarin.Forms 5.0.0.2612 requires Xamarin.Google.Android.Material (>= 1.4.0.2 && < 1.9.0) but version Xamarin.Google.Android.Material 1.10.0.1 was resolved.
NU1608: Detected package version outside of dependency constraint: Xamarin.Forms 5.0.0.2612 requires Xamarin.AndroidX.Browser (>= 1.3.0.6 && < 1.6.0) but version Xamarin.AndroidX.Browser 1.7.0 was resolved.
NU1608: Detected package version outside of dependency constraint: Xamarin.Forms 5.0.0.2612 requires Xamarin.AndroidX.Navigation.UI (>= 2.3.5.1 && < 2.6.0) but version Xamarin.AndroidX.Navigation.UI 2.7.5 was resolved.
```

### Basic Information

- Version with issue: 5.0.0.2612
- Last known good version: N/A
- Platform Target Frameworks: 
  - iOS:  
  - Android: Android API 33
  - UWP:   
- Android Support Library / AndroidX Version: 
- NuGet Packages:
- Affected Devices:

### Environment

<details><summary>Show/Hide Visual Studio info</summary>

```
Version 17.8.1
VisualStudio.17.Release.LTSC.17.8/17.8.1+34316.72
Microsoft .NET Framework
Version 4.8.09032

Xamarin   17.8.0.155 (d17-8@7042beb)
Visual Studio extension to enable development for Xamarin.iOS and Xamarin.Android.

Xamarin Designer   17.8.3.6 (remotes/origin/d17-8@eccf46a291)
Visual Studio extension to enable Xamarin Designer tools in Visual Studio.

Xamarin Studio Linker 2022   1.3.2
Enables Visual Studio users to edit their Xamarin 'Resource' files (such as layouts and vector drawables) in Android Studio, directly reflecting the changes inside their Visual Studio project.

Xamarin Templates   17.8.16 (830b56a)
Templates for building iOS, Android, and Windows apps with Xamarin and Xamarin.Forms.

Xamarin.Android SDK   13.2.2.0 (d17-5/45b0e14)
Xamarin.Android Reference Assemblies and MSBuild support.
    Mono: d9a6e87
    Java.Interop: xamarin/java.interop/d17-5@149d70fe
    SQLite: xamarin/sqlite/3.40.1@68c69d8
    Xamarin.Android Tools: xamarin/xamarin-android-tools/d17-5@ca1552d


Xamarin.iOS and Xamarin.Mac SDK   16.4.0.23 (9defd91b3)
Xamarin.iOS and Xamarin.Mac Reference Assemblies and MSBuild support.
```

</details>

**Comment by jfversluis** [comment]
Yeah we tend to be a bit strict about the versions of the dependencies since we have been bitten by that before and things broke. I'll see if I can update them a little bit and see if that still works OK. Would you be willing to test the packages that are generated by a PR from this to see if that still works good for your app?

**Comment by MagicAndre1981** [comment]
my apps works after updating the androidx references. Only the warnings are annoying.

**Comment by MagicAndre1981** [comment]
Also update `Xamarin.AndroidX.Preference`, the [Version 1.1.1.9 you use](https://github.com/xamarin/Xamarin.Forms/blob/88b70ad2824a4f40fa2ef056439ada9dc3e85ab4/.nuspec/Xamarin.Forms.nuspec#L28) is [Monodroid9 and not monodroid13](https://www.nuget.org/packages/Xamarin.AndroidX.Preference/1.1.1.9#supportedframeworks-body-tab) and may cause issues.

**Comment by jfversluis** [comment]
I've thought about this one for a future release to not do the restraints from our side but I decided to keep them in place. If I took the version range off it might cause people to just install the latest versions and things might break over time. With this still in place, we can sort of safely say that it has been stable with the versions that we have indicated here, even after the Xamarin.Forms support has expired.

If someone still wants to update to the latest packages they will get these warnings and you're free to ignore them or even not see them anymore if that's what you want by suppressing them.

I will bump that Preference one although I think it's fine but better safe than sorry.

Thanks!
